src/OpenAPM.ts

```diff
@@ -273,9 +287,14 @@ export class OpenAPM extends LevitateEvents {
       }
 
       // Make sure you copy baseURL in case of nested routes.
-      const path = req.route ? req.baseUrl + req.route?.path : pathname;
+      const path = req.route
+        ? req.route?.path !== '*'
+          ? req.baseUrl + req.route?.path
+          : pathname
+        : pathname;
+
```

Comment:
@chtushar This will not be needed now, correct?

Reply:
We can keep it. There could be cases where users are using express and has a handler for `all`.

Reply:
Yes but let's add it in a separate PR.